File: Weld.Test/ViewModels/SimpleViewModel.ts
/// <reference path="../typings/jquery/jquery.d.ts" />
class SimpleViewModel 
{
    static Enabled: string = "Enabled";
    static get $Enabled() : JQuery {
        return $("#Enabled");
    }
    static FirstName: string = "FirstName";
    static get $FirstName() : JQuery {
        return $("#FirstName");
    }
}

File: Weld.Test/ViewModels/SimpleViewModel.cs
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Weld.Attributes;

namespace Weld.Test.ViewModels
{
    [WeldViewModel]
    class SimpleViewModel
    {
        public bool Enabled { get; set; }

        public string FirstName { get; set; }
        
    }
}
